ipn/localapi/localapi.go

```diff
@@ -967,6 +968,23 @@ func (h *Handler) serveCheckIPForwarding(w http.ResponseWriter, r *http.Request)
 	})
 }
 
+func (h *Handler) serveCheckUDPGROForwarding(w http.ResponseWriter, r *http.Request) {
+	if !h.PermitRead {
+		http.Error(w, "UDP GRO forwarding check access denied", http.StatusForbidden)
+		return
+	}
+	var warning string
+	if err := h.b.CheckUDPGROForwarding(); err != nil {
+		warning = err.Error()
+	}
```

Comment:
so it starts with (warning, error) which gets mushed into (error) which now gets mushed into (warning)

But sure :)

Reply:
I think it's appropriate for the JSON field to be Warning and not Error here. Agree that what's further up is a confusing dance.